packages/kit/src/views/Onboarding/pages/ConnectHardwareWallet/ConnectYourDevice.tsx

```
@@ -1736,13 +1739,13 @@ export function ConnectYourDevicePage() {
               },
               ...(platformEnv.isSupportDesktopBle
                 ? [
-                    {
-                      label: intl.formatMessage({
-                        id: ETranslations.global_bluetooth,
-                      }),
-                      value: EConnectDeviceChannel.bluetooth,
-                    },
-                  ]
+                  {
```

ESLint

[error] Insert `··`

(prettier/prettier)